nasa/earthdata-search
# EDSC-3416: Implementing collection results style changes

# Overview

### What is the feature?

Adds the cloud hosted flag and implements an A/B test of various elements in the collection search results

### What is the Solution?

Display the flag

### What areas of the application does this impact?

Collection search results

# Testing

### Reproduction steps

- **Environment for testing:** SIT
- **Collection to test with:** Any

1. Load a search for a collection with the cloud hosted flag enabled
2. Confirm the cloud hosted flag is displayed
3. Confirm no issues with new layout

# Checklist
